lib/index.js

```diff
@@ -24,13 +24,13 @@ internals.schema = Joi.object({
     cookie: Joi.string().default('sid'),
     password: Joi.string().required(),
     ttl: Joi.number().integer().min(0).when('keepAlive', { is: true, then: Joi.required() }),
-    domain: Joi.string(),
+    domain: Joi.string().allow(null),
     path: Joi.string().default('/'),
     clearInvalid: Joi.boolean().default(false),
     keepAlive: Joi.boolean().default(false),
     isSecure: Joi.boolean().default(true),
     isHttpOnly: Joi.boolean().default(true),
-    redirectTo: Joi.string(),
+    redirectTo: Joi.string().allow(false),
```

Comment:
I used false to be consistent with the docs for redirectTo at the route level:

```
 To set an individual route to use or disable redirections, use the route plugins config ({ config: { plugins: { 'hapi-auth-cookie': { redirectTo: false } } } }). Defaults to no redirection.
```

In light of that, would you still like me to change it?